src/app_service/service/node_configuration/service.nim

```diff
@@ -144,9 +135,17 @@ proc setWakuVersion*(self: Service, wakuVersion: int): bool =
   let newConfiguration = setWakuConfig(self.configuration, wakuVersion)
   return self.saveConfiguration(newConfiguration)
 
-method isCommunityHistoryArchiveSupportEnabled*(self: Service): bool =
+proc isCommunityHistoryArchiveSupportEnabled*(self: Service): bool =
   return self.configuration.TorrentConfig.Enabled
 
+proc enableCommunityHistoryArchiveSupport*(self: Service): bool =
+  let response = status_node_config.enableCommunityHistoryArchiveSupport()
+  if(not response.error.isNil):
+    error "error enabling community history archive support: ", errDescription = response.error.message
```

Comment:
One thing to consider: lots of the code you're removing here was recently introduced to have better error handling and user experience when enabling the archive protocol has failed: https://github.com/status-im/status-desktop/pull/8354

While we log errors to the console. in release builds people will not see the error message anymore.
I expect developers to run into this (when they run Status release + a dev build and both use the same torrent port).

Reply:
Good point. I'm hoping that it's not gonna happen because most dev accounts are gonna be new account (so this is not called).
Also, if it's dev accounts, it's not a big deal because it's an edge case and it's not a road blocking setting.